homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -116,28 +116,34 @@ def _update_info(self):
         """
         _LOGGER.debug("Checking ARP")
 
-        endpoint = "Wireless" if self.wireless_only else "Lan"
-        url = f"{self.protocol}://{self.host}/Status_{endpoint}.live.asp"
-        data = self.get_ddwrt_data(url)
+        url = f"{self.protocol}://{self.host}/Status_Wireless.live.asp"
```

Comment:
This a an “else” to the “if” in line 122.

Reply:
No. The original logic was "if (false) do A else do B", while it should have been. "Do A. If (false) also do B."

We always want to get the wireless status page. And if "wireless_only" is set to "false", we also get the "Lan" status page.

Reply:
ok